rust/src/lib.rs

```diff
@@ -862,29 +862,47 @@ pub unsafe extern "C" fn zcashlc_list_transparent_receivers(
     db_data_len: usize,
     account_id: i32,
     network_id: u32,
+    include_ephemeral: bool,
```

Comment:
Also this doesn't expose the [`index_range` parameter](https://github.com/zcash/librustzcash/blob/c6db09b25a628b0b60ab3c0e0f5bd3880d6b5b3a/zcash_client_backend/src/data_api.rs#L1211-L1212) of `get_known_ephemeral_addresses`. If it were a separate method then it would be natural to do that.

Reply:
The way I've been approaching this is that breaking changes go to `main`, semver-compatible changes go to a `hotfix/` branch. I thought that's what we had agreed upon?

Reply:
We had agreed upon that for regular repositories, but this repository is unique in its committed binary requirement. If we merge breaking changes directly to `main`, then we have the problem that `main` has a split view: its code does not match its binaries, and it's the latter that actually get used.

If we want to have a primary development target branch rather than creating a separate feature branch for each release like we've done in the past, then we should do so explicitly:
- Have a `dev` branch that is explicitly a split view branch, where the binary is never updated.
- We cut breaking releases by making a branch from `dev`, building the binaries, and merging that branch into `main` via a PR.
- For testing the current state of the `dev` branch, we could have a `dev-preview` branch that is maybe built by CI (if performant enough) that is force-pushed every time `dev` changes (so `dev-preview` only ever contains a single commit on top of the current state of `dev`, ensuring that old preview binaries can get GCed).

Reply:
Okay, that makes sense, I had forgotten about the API/binary split issue. We should have a feature branch both for a semver-compatible hotfix, and also for the next semver-breaking release, and we should target each PR to one or both respectively. https://github.com/Electric-Coin-Company/zcash-light-client-ffi/pull/158 and this PR should both be retargeted (and #158 can have a separate PR opened to the hotfix branch, if you prefer.)

We can't actually build the preview in CI because we have to build on XCode 13 to meet our iOS version compatibility requirements in order to avoid breaking Edge's process, and I don't think we can get a MacOS runner that is on an old enough version to still run XCode 13?

Reply:
Opened #162.